lib/vre/ch31_form.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require_relative 'service'
+require 'sentry_logging'
+
+module VRE
+  class Ch31Form < Service
+    include SentryLogging
+    class VRECH31Error < StandardError
```

Comment:
Move this into it's own home at `lib/vre/errors/ch31_error.rb` or something similar